rust/src/rfb/parser.rs

```diff
@@ -24,6 +24,7 @@ use nom7::combinator::map_res;
 use nom7::number::streaming::*;
 use nom7::*;
 
+#[derive(Copy, Clone, PartialOrd, PartialEq, Eq, Debug)]
```

Comment:
Which operations does this PR do to use these?

Reply:
I was getting errors for PartialEq and Debug.

Reply:
Following up on Shivani's question - do we need all of those?